python/libraft/pyproject.toml

```diff
@@ -31,6 +31,7 @@ authors = [
 license = { text = "Apache 2.0" }
 requires-python = ">=3.10"
 dependencies = [
+    "librmm==25.4.*,>=0.0.0a0",
```

Comment:
I'd expected to see direct runtime and build-time dependencies on `rapids-logger` in `libraft` wheels and the conda recipe, as was done for `libcudf` in https://github.com/rapidsai/cudf/pull/17899, because RAFT is exposing its own logger.

Don't we need that?

Reply:
Yes, we do. That is a good call, I'll add those.

Reply:
Wow I really skated by on transitive dependencies in this repo. Fixed in https://github.com/rapidsai/raft/pull/2566/commits/a0bb265be07f60a61c5259ddeeb3a100b9731575.